### DISCUSSION TITLE
--name 无法使用中文

### DISCUSSION THREAD

**Discussion by kewillcai** [tw93/Pake#626] [Q&A] [answered]
当我 --name 使用中文时，就会报错

✕ Name should only include letters and numbers, and must contain at least one letter. Examples: 123pan, 123Pan, Pan123, weread, WeRead, WERead.

用网页打包也是打包不了， 是不支持中文吗，请问有什么好的解决办法吗？

**Comment by Tlntin** [comment, marked as answer]
不支持
